### PR TITLE
Updated the cryptosuite sections, mostly the Selective Disclosure part

### PR DESCRIPTION
**Pull request by iherman**
Hopefully I got it right now...

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-overview/pull/16.html" title="Last updated on Oct 17, 2024, 12:59 PM UTC (0869af9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-overview/16/73c2c35...0869af9.html" title="Last updated on Oct 17, 2024, 12:59 PM UTC (0869af9)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -910,20 +915,36 @@ <h5>Selective Disclosure Schemes</h5>
...
-              Furthermore, each non-mandatory claim must be signed individually, yielding a series of signatures.
-              The final Base Proof is, conceptually, the concatenation of all these signatures and related informations like the separation of mandatory and non-mandatory claims.
+              To calculate the base proof, the Credential is supplemented with extra information that separates the <em>mandatory</em> and <em>non-mandatory</em> claims.
+              Using that information, the issuer's transformation step in the <a href="#di-integrity-structure">general data integrity structure</a> produces a
+              data structure containing the hash of the proof options (much like in the case of the full disclosure schemes) and of the set of mandatory claims, plus the identification of mandatory claims. 
+              Some elements of this data structure are then signed, the resulting structure is converted into CBOR, and 
+              encoded to provide as a Multikey formatted `proofValue`.
```
Comment (dlongley) [inline index.html:926]:
```suggestion
              encoded to provide as a multibase-formatted `proofValue`. 
```

Comment (iherman) [reply index.html:926]: Oops...

**Review by Wind4Greg — CHANGES_REQUESTED** [review]
Hi Ivan, looks pretty good but a couple things are confusing to read or not quite accurate. (1) You use the term "[general data integrity structure](https://pr-preview.s3.amazonaws.com/w3c/vc-overview/pull/16.html#di-integrity-structure) when it seems that this is a "procedure" that produces a "structure". (2) The use of a "challenge" is not specific to SD schemes, and in fact BBS would use a different approach to achieving the same thing.

**Comment by iherman** [comment]
Thank you @Wind4Greg 

> (1) You use the term "[general data integrity structure](https://pr-preview.s3.amazonaws.com/w3c/vc-overview/pull/16.html#di-integrity-structure) when it seems that this is a "procedure" that produces a "structure". 

Right. This was the result of a poor choice for the title of section 4.2.1.; I have changed that and made the changes in the references. I hope it is better now.

> (2) The use of a "challenge" is not specific to SD schemes, and in fact BBS would use a different approach to achieving the same thing.

I removed the term and reformulated/removed the sentences that used it.

I hope it is better now!

File: index.html [hunk 2]
```diff
@@ -312,7 +312,7 @@ <h2>High Level View of the Specifications</h2>
...
...iagrams/VCWG_specifications.svg" alt="Diagram illustrating the structure of the Verifiable Credentials (VCs) specification, using boxes labeled with references to the specification. Arrows on the diagram connect some of the boxes to represent (normative) dependencies. The box labeled as the VC Data Model is at the center of the diagram. Additional component boxes, such as JSON Schema for structural checking, Bitstring Status List for publishing status information, and Controller Documents for verification methods and relationships, are related to the VC Data Model. Boxes for securing mechanisms represent separate groups of boxes, with further grouping for embedded proofs and enveloping proofs. The group for embedded proofs includes a box labeled as Data Integrity, which depends on the VC Data Model and the Controller Document; and the ECDSA, EdDSA, and BBS Cryptosuites for different elliptic curves and signature schemes, which all depend on Data Integrity. Furthermore, BBS Cryptosuite also depends on ECDSA. The enveloped proofs section includes a single box labeled as JOSE and COSE using JWS, SD-JWT, or CBOR, and is also dependent on the VC Data Model and the Controller Document.">
```
[the remark this answers is not in the document]
Comment (iherman) [reply index.html:315]: See https://github.com/w3c/vc-overview/pull/17. The PR was closed by the time you made this comment...

**Pull request merged**